FSharpWorkspaceShim/FSharpWorkspaceShim.fsproj

```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="FSharp.Compiler.Service" />
+    <PackageReference Include="FSharp.Compiler.Service" Version="31.0.0" />
```

Comment:
Make this change in `Directory.Build.targets`.

Reply:
Done